server/api_link.go

```diff
@@ -441,16 +442,18 @@ func (s *ApiServer) LinkGoogle(ctx context.Context, in *api.AccountGoogle) (*emp
 		s.logger.Info("Could not authenticate Google profile.", zap.Error(err))
 		return nil, status.Error(codes.Unauthenticated, "Could not authenticate Google profile.")
 	}
-	
+
 	displayName := googleProfile.Name
 	if len(displayName) > 255 {
-		// Trim the name in case it is longer than db can store
-		displayName = displayName[0:255]		
+		// Ignore the name in case it is longer than db can store
+		s.logger.Warn("Skipping updating display_name: value received from Google longer than max length of 255 chars.", zap.String("display_name", displayName))
+		displayName = ""
 	}
-	
+
 	avatarUrl := googleProfile.Picture
-	if len(avatarUrl) > 512 {
+	if len(avatarUrl) > 512 || avatarUrl == "" {
```

Comment:
Nitpicking - Don't think we need the `|| avatarUrl == ""` in the if statement? 

If we do, we need to update the log message to say "value is either empty or longer than max length of 512 chars."